src/AutoRest.CSharp/Common/Input/InputTypes/OperationResponse.cs

```diff
@@ -3,10 +3,30 @@
 
 using System;
 using System.Collections.Generic;
+using System.Linq;
+using AutoRest.CSharp.Utilities;
 
 namespace AutoRest.CSharp.Common.Input;
 
-internal record OperationResponse(IReadOnlyList<int> StatusCodes, InputType? BodyType, BodyMediaType BodyMediaType, IReadOnlyList<OperationResponseHeader> Headers, bool IsErrorResponse)
+internal record OperationResponse(IReadOnlyList<int> StatusCodes, InputType? BodyType, BodyMediaType BodyMediaType, IReadOnlyList<OperationResponseHeader> Headers, bool IsErrorResponse, string? ResultPath)
 {
-    public OperationResponse() : this(StatusCodes: Array.Empty<int>(), BodyType: null, BodyMediaType: BodyMediaType.None, Headers: Array.Empty<OperationResponseHeader>(), IsErrorResponse: false) { }
+    public OperationResponse() : this(StatusCodes: Array.Empty<int>(), BodyType: null, BodyMediaType: BodyMediaType.None, Headers: Array.Empty<OperationResponseHeader>(), IsErrorResponse: false, ResultPath: null) { }
```

Comment:
`ResultPath` is meaningful only in LRO. Can we move this property to the `OperationLongRunning`?

Reply:
done